crates/ra_ide/src/syntax_highlighting.rs

```diff
@@ -110,9 +112,17 @@ pub(crate) fn highlight(
                 _ => token.into(),
             }
         } else {
-            element
+            element.clone()
         };
 
+        if let Some(token) = element.as_token().cloned().and_then(ast::RawString::cast) {
+            let expanded = element_to_highlight.as_token().unwrap().clone();
+            if highlight_injection(&mut res, &sema, token, expanded).is_some() {
+                eprintln!("res = {:?}", res);
```

Comment:
Is this a leftover?

Reply:
yup, I've removed this during the weekend I think